bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -52,7 +61,7 @@ logging.info("Clustering coinc triggers (exclusive of zerolag)")
 e = e.cluster(args.cluster_window)
 
 logging.info("Dumping foreground triggers")
-f = h5py.File(args.output_file, "w")
+f = h5py.fw(args.output_file)
```

Comment:
Ok, you are right. I need to check over my test to make sure it actually did what I thought it did.